### ISSUE TITLE
markdown table with a indentation displays unsuccessfully on website.

### ISSUE THREAD

**Issue by kwilove** [hexojs/hexo#3560]
## Environment Info

Node version(`node -v`: **Should be at least nodejs 6.9. If not, please update before posting an issue!**):  v10.15.1

Your site `_config.yml` (Optional):

Your theme `_config.yml` (Optional):    yalia

Hexo and Plugin version(`npm ls --depth 0`): 

Your package.json `package.json`: 

## It display unsuccessfully markdown table with a indentation

markdown allows the table that adds a indentation, but this table displays unsuccessfully on website after executing `hexo deploy` command。

For example:
![image](https://user-images.githubusercontent.com/13587047/58304357-75b27f00-7e26-11e9-92c4-50e161cdedaf.png)

Display on website:
![image](https://user-images.githubusercontent.com/13587047/58304275-29673f00-7e26-11e9-907e-f24b140f9ffe.png)

**How should I solve this problem?**

**Comment by YoshinoriN** [comment]
@kwilove 
Because "4 spaces" means code block in a markdown specification.
Please see below.

* [GFM example](https://github.github.com/gfm/#example-107)
* [code-block](https://markdown-guide.readthedocs.io/en/latest/basics.html#code-block)
* [Why does markdown use 4-space indent instead of code?](https://meta.stackexchange.com/questions/111202/why-does-markdown-use-4-space-indent-instead-of-code-or-code)

**Comment by kwilove** [comment]
Thank you for your reply!
But i have a new question:

use the same markdown text:
![image](https://user-images.githubusercontent.com/13587047/58328888-d319f080-7e65-11e9-8db3-3b7e02fa09a5.png)

And then i open directly previous markdown file by browser, it will displays:
![image](https://user-images.githubusercontent.com/13587047/58328999-1b391300-7e66-11e9-8203-cbf6745322c0.png)

Why does it work on browser successfully?

**Comment by YoshinoriN** [comment]
@kwilove 
Do you use a something extension to a browser?
My understanding, a browser cannot be rendering a markdown.

**Comment by kwilove** [comment]
@YoshinoriN 
Yes, you are right.
I using a IDE called "Visual Studio Code" to write a markdown file, and the IDE has installed a plugin "Markdown Preview Enhanced", is this the cause?

**Comment by YoshinoriN** [comment]
@kwilove 
OK. I got it. It is not VSCode plugins problem. Do you using [hexo-renderer-marked](https://github.com/hexojs/hexo-renderer-marked)?
hexo-renderer-marked is depend on marked. It seems fixed in [markd #1446](https://github.com/markedjs/marked/pull/1446) and it's released in v0.6.2

Please see below demo.

* [v0.6.1](https://marked.js.org/demo/?text=1.%20success%3A%0A%0A%7Ctest%7Ctest%7C%0A%7C---%7C---%7C%0A%7Ca%7Cb%7C%0A%7Ca%7Cb%7C%0A%0A2.%20fail%3A%0A%0A%20%20%20%20%7Ctest%7Ctest%7C%0A%20%20%20%20%7C---%7C---%7C%0A%20%20%20%20%7Ca%7Cb%7C%0A%20%20%20%20%7Ca%7Cb%7C&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=0.6.1) Broken
* [v0.6.2](https://marked.js.org/demo/?text=1.%20success%3A%0A%0A%7Ctest%7Ctest%7C%0A%7C---%7C---%7C%0A%7Ca%7Cb%7C%0A%7Ca%7Cb%7C%0A%0A2.%20fail%3A%0A%0A%20%20%20%20%7Ctest%7Ctest%7C%0A%20%20%20%20%7C---%7C---%7C%0A%20%20%20%20%7Ca%7Cb%7C%0A%20%20%20%20%7Ca%7Cb%7C&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=0.6.2) Fixed

We [already bump to marked  0.6.x](https://github.com/hexojs/hexo-renderer-marked/pull/87) at [hexo-renderer-marked v1.0.0](https://github.com/hexojs/hexo-renderer-marked/releases/tag/1.0.0)

Would you try to update your hexo-renderer-marked to 1.0.1?
Please be careful it has some [breaking changes](https://github.com/hexojs/hexo-renderer-marked/releases/tag/1.0.0).

Below is dependency about hexo-renderer-marked 1.0.1

```
C:\Users\N.Yoshinori\tmp\marked-test
`-- hexo-renderer-marked@1.0.1
  +-- hexo-util@0.6.3
  | +-- bluebird@3.5.4
  | +-- camel-case@3.0.0
  | | +-- no-case@2.3.2
  | | | `-- lower-case@1.1.4
  | | `-- upper-case@1.1.3
  | +-- cross-spawn@4.0.2
  | | +-- lru-cache@4.1.5
  | | | +-- pseudomap@1.0.2
  | | | `-- yallist@2.1.2
  | | `-- which@1.3.1
  | |   `-- isexe@2.0.0
  | +-- highlight.js@9.15.6
  | +-- html-entities@1.2.1
  | `-- striptags@2.2.1
  +-- marked@0.6.2
  `-- strip-indent@2.0.0
```

I'm so sorry to wrong answer when first time...

**Comment by kwilove** [comment]
@YoshinoriN 
Well，i will try it，thanks.
